ninfod/ninfod_core.c

```diff
@@ -158,11 +158,11 @@ static __inline__ struct subjinfo *subjinfo_lookup(size_t code)
 #define QTYPEINFO_F_RATELIMIT	0x1
 
 struct qtypeinfo {
-	uint16_t qtype;
 	char	*name;
 	int	(*getreply)(CHECKANDFILL_ARGS);
 	void	(*init)(INIT_ARGS);
 	int	flags;
+	uint16_t qtype;
 };
 
 static struct qtypeinfo qtypeinfo_table[] = {
```

Comment:
I don't understand compilation enough to ack this. https://reviews.llvm.org/D14779 shows this warning is noisy. If really improvement, I'd also move `.qtype =` to the end to not to be confusing.

Reply:
It's an exercise in struct packing.

Reply:
Right, acked this one :).
http://www.catb.org/esr/structure-packing/